### ISSUE TITLE
Not able to run the Paperless app in emulator and can't build an apk

### ISSUE THREAD

**Issue by Ghostsmaw** [bauerj/paperless_app#50]
Here is the error am getting when trying to build the paperless app after downloading the source code:

Error Code is bellow
<img width="1680" alt="Screenshot 2021-09-06 at 18 47 03" src="https://user-images.githubusercontent.com/25077504/132250211-4a8d9211-2889-4444-8f59-cc9815dae157.png">
<img width="1680" alt="Screenshot 2021-09-06 at 18 47 10" src="https://user-images.githubusercontent.com/25077504/132250325-70c7b722-ba7f-41e7-8787-2cc39033b282.png">

../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:572:9: Error: The setter 'tristate' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'tristate'.
      ..tristate = tristate
        ^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:574:9: Error: The setter 'activeColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'activeColor'.
      ..activeColor = activeColor
        ^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:576:9: Error: The setter 'inactiveColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'inactiveColor'.
      ..inactiveColor = inactiveColor
        ^^^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:577:9: Error: The setter 'focusColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'focusColor'.
      ..focusColor = focusColor
        ^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:578:9: Error: The setter 'hoverColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'hoverColor'.
      ..hoverColor = hoverColor
        ^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:579:9: Error: The setter 'reactionColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'reactionColor'.
      ..reactionColor = reactionColor
        ^^^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:580:9: Error: The setter 'inactiveReactionColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'inactiveReactionColor'.
      ..inactiveReactionColor = inactiveReactionColor
        ^^^^^^^^^^^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:581:9: Error: The setter 'splashRadius' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'splashRadius'.
      ..splashRadius = splashRadius
        ^^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:582:9: Error: The setter 'onChanged' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'onChanged'.
      ..onChanged = onChanged
        ^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:583:9: Error: The setter 'additionalConstraints' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'additionalConstraints'.
      ..additionalConstraints = additionalConstraints
        ^^^^^^^^^^^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:584:9: Error: The setter 'vsync' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'vsync'.
      ..vsync = vsync
        ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:585:9: Error: The setter 'hasFocus' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'hasFocus'.
      ..hasFocus = hasFocus
        ^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:586:9: Error: The setter 'hovering' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'hovering'.
      ..hovering = hovering
        ^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:616:11: Error: No named parameter with the name 'value'.
          value: value,
          ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:643:11: Error: Superclass has no setter named 'value'.
    super.value = newValue;
          ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:639:21: Error: The getter 'value' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'value'.
    if (newValue == value) {
                    ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:642:17: Error: The getter 'value' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'value'.
    _oldValue = value;
                ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:648:11: Error: Superclass has no method named 'describeSemanticsConfiguration'.
    super.describeSemanticsConfiguration(config);
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:649:24: Error: The getter 'value' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'value'.
    config.isChecked = value == true;
                       ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:669:11: Error: The getter 'activeColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'activeColor'.
        ? activeColor
          ^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:670:22: Error: The getter 'inactiveColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'inactiveColor'.
        : Color.lerp(inactiveColor, activeColor, t * 4.0)!;
                     ^^^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:670:37: Error: The getter 'activeColor' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'activeColor'.
        : Color.lerp(inactiveColor, activeColor, t * 4.0)!;
                                    ^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:727:5: Error: The method 'paintRadialReaction' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing method, or defining a method named 'paintRadialReaction'.
    paintRadialReaction(canvas, offset, size.center(Offset.zero));
    ^^^^^^^^^^^^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:727:41: Error: The getter 'size' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'size'.
    paintRadialReaction(canvas, offset, size.center(Offset.zero));
                                        ^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:731:19: Error: The getter 'size' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'size'.
        offset + (size / 2.0 - const Size.square(_kEdgeSize) / 2.0 as Offset);
                  ^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:732:36: Error: The getter 'position' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'position'.
    final AnimationStatus status = position.status;
                                   ^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:735:15: Error: The getter 'position' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'position'.
            ? position.value
              ^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:736:21: Error: The getter 'position' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'position'.
            : 1.0 - position.value;
                    ^^^^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:739:31: Error: The getter 'value' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'value'.
    if (_oldValue == false || value == false) {
                              ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:740:24: Error: The getter 'value' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'value'.
      final double t = value == false ? 1.0 - tNormalized : tNormalized;
                       ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:752:34: Error: The getter 'value' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'value'.
        if (_oldValue == null || value == null)
                                 ^^^^^
../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart:771:13: Error: The getter 'value' isn't defined for the class '_RenderCheckbox'.
 - '_RenderCheckbox' is from 'package:wechat_assets_picker/src/widget/custom_checkbox.dart' ('../../desktop/tools/flutter/.pub-cache/hosted/pub.dartlang.org/wechat_assets_picker-5.1.0/lib/src/widget/custom_checkbox.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'value'.
        if (value == true)
            ^^^^^

FAILURE: Build failed with an exception.

* Where:
Script '/Users/williamspeter/Desktop/tools/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 1035

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command '/Users/williamspeter/Desktop/tools/flutter/bin/flutter'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

**Comment by bauerj** [comment]
Can you post the output of `flutter doctor`?

**Comment by Ghostsmaw** [comment]
> Can you post the output of `flutter doctor`?

<img width="1680" alt="Screenshot 2021-09-06 at 21 59 18" src="https://user-images.githubusercontent.com/25077504/132260247-037c724e-f2d0-4717-8c6c-94d2cd183fdc.png">
flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.2.3, on Mac OS X 10.15.7 19H1217 darwin-x64, locale en-NG)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[!] Xcode - develop for iOS and macOS
    ✗ Xcode installation is incomplete; a full installation is necessary for iOS development.
      Download at: https://developer.apple.com/xcode/download/
      Or install Xcode via the App Store.
      Once installed, run:
        sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
        sudo xcodebuild -runFirstLaunch
[✓] Android Studio (version 4.1)
[✓] Connected device (1 available)

**Comment by bauerj** [comment]
Thanks, I'll try to reproduce the issue with the versions you have.

**Comment by Ghostsmaw** [comment]
> Thanks, I'll try to reproduce the issue with the versions you have.

okay thank you very much, i will be on standby

**Comment by qcasey** [comment]
I also ran into this when building. It is introduced with the newer version of flutter, breaking `flutter_wechat_assets_picker` which is [described and fixed here](https://github.com/fluttercandies/flutter_wechat_assets_picker/pull/131).

To fix, I updated to the oldest compatible package in pubspec.yaml:

```yaml
  wechat_assets_picker: ^5.4.0
```

I also had to bump the kotlin version to latest in our build.gradle:

```gradle
    ext.kotlin_version = '1.5.10'
```

Which seemed to break `edge_detection`. I made a [few minor changes](https://github.com/qcasey/edge_detection/commits/master) along with a local NDK update which finally allowed me to build `paperless_app`. I don't know kotlin or the implications but it appears to still work. 

@bauerj I can create some PRs for these, if that makes things easy?

![Screenshot_20210913-103647_Paperless](https://user-images.githubusercontent.com/1429672/133130822-9489a8f8-ca47-4613-af59-1cb9ede120bd.png)
